src/main/java/uk/gov/register/configuration/RegisterContentPages.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.register.configuration;
+
+import java.util.Optional;
+
+public class RegisterContentPages {
+    private final Optional<String> registerHistoryUrl;
+
+    public RegisterContentPages(Optional<String> registerHistoryUrl) {
+        this.registerHistoryUrl = registerHistoryUrl;
+    }
+
+    @SuppressWarnings("unused, used from template")
+    public String getRegistersIntroductionUrl() {
+        return "https://www.gov.uk/government/publications/registers/registers";
+    }
+
+    @SuppressWarnings("unused, used from template")
+    public String getUsingRegistersGuidanceUrl() {
+        return "https://www.gov.uk/guidance/using-registers-to-build-a-service";
+    }
+
+    @SuppressWarnings("unused, used from template")
+    public String getSpecificationUrl() {
+        return "http://openregister.github.io/specification/";
+    }
+
+    @SuppressWarnings("unused, used from template")
+    public String getRegisterHistoryPageUrl() {
```

Comment:
Can we just provide the `Optional` direct to the template and have the template call `isPresent` and `get`? That's a pattern we do elsewhere - see `copyright.html` for an example